Bottom: 144d4a9dcb2e56877dc4e5ba7240c68f494d842c
Top:    70fc3cc451704c5e5c7f21f859ffc20940d5212f
Author: scott snyder <sss@karma>
Date:   2020-02-01 10:37:02 -0500

gitignore additions


---

diff --git a/.gitignore b/.gitignore
index d9d3967a12c..dd80578383e 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+NOTES
 *.diff
 *.patch
 *.orig
